<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HNCloudWidget</class>
 <widget class="QWidget" name="HNCloudWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetCloud" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>40</number>
      </property>
      <item row="2" column="2">
       <widget class="HNPushButton" name="pushButton_del">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="6">
       <widget class="HNPushButton" name="btn_print">
        <property name="text">
         <string>Print</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="HNPushButton" name="pushButton_down">
        <property name="text">
         <string>Down</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="HNPushButton" name="pushButton_upload">
        <property name="text">
         <string>Upload</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>59</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="7">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>59</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>59</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="8">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="HNCloudTreeWidget" name="treeCloud"/>
         </item>
         <item>
          <widget class="HNCloudLocalTreeWidget" name="treeLocal"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Cloud Service:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HNPushButton</class>
   <extends>QPushButton</extends>
   <header>hnpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>HNCloudTreeWidget</class>
   <extends>QTreeView</extends>
   <header>hncloudtreewidget.h</header>
  </customwidget>
  <customwidget>
   <class>HNCloudLocalTreeWidget</class>
   <extends>QTreeView</extends>
   <header>hncloudlocaltreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>heartbeatclick()</slot>
  <slot>getdirclick()</slot>
  <slot>getfilesclick()</slot>
  <slot>upload()</slot>
 </slots>
</ui>
